### PR TITLE
add map marker by community location & add 1200x630 og logo

### LINKED ISSUE #32: Task: Add community map
add a map displaying various communities present in India

### PR DESCRIPTION
**Pull request by bharattech**
add map marker by community locations

theme template modified

add 1200x630 og logo

Fixes #32 and #29

### CODE REVIEW

**Review by me-diru — COMMENTED** [review]
kindly make the changes requested. Few more I would like to add:

1) Can you try using a custom marker using Ferris happy image found at https://rustacean.net/ https://leafletjs.com/examples/custom-icons/

2) Can we make the map fixed/immovable outside India, the frame size is right. maybe as shown here in second example: https://www.semantic-mediawiki.org/wiki/Maps_examples/Leaflet_with_a_static_map

3) can we keep the map above the table with all the socials? and make the map a bit smaller so that the table is visible when the page loads. I think the visual makes it more appealing

Thank you for contributing, looking forward to final verion :D

File: templates/docs/map.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+          }).addTo(map);
+          const indiaMarker = L.marker([20.5937, 78.9629]).addTo(map);
+          indiaMarker.bindPopup("<b>Hello world!</b><br>Welcome to <b>Rust India.</b>").openPopup();
+          const puneMarker = L.marker([18.5204, 73.8567]).addTo(map);
+          puneMarker.bindPopup("<b>Hello world!</b><br>Welcome to <b>Rust Pune.</b>");
+          const keralaMarker = L.marker([10.1632, 76.6413]).addTo(map);
+          keralaMarker.bindPopup("<b>Hello world!</b><br>Welcome to <b>Rust Kerala.</b>");
+          const bengaluruMarker = L.marker([12.9716, 77.5946]).addTo(map);
+          bengaluruMarker.bindPopup("<b>Hello world!</b><br>Welcome to <b>Rust Bengaluru.</b>");
+          const hyMarker = L.marker([17.3850, 78.4867]).addTo(map);
```
[the remark this answers is not in the document]
Comment (bharattech) [reply templates/docs/map.html:45]: Yes sure! ( I Love "Hyderabad" )

File: templates/docs/map.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        
+
+        <div id="map"></div>
+        <script>
+          const map = L.map('map').setView([20.5937, 78.9629], 5);
+          L.tileLayer('https://tile.openstreetmap.org/{z}/{x}/{y}.png', {
+            maxZoom: 19,
+            attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>'
+          }).addTo(map);
+          const indiaMarker = L.marker([20.5937, 78.9629]).addTo(map);
```
[the remark this answers is not in the document]
Comment (bharattech) [reply templates/docs/map.html:37]: Okay

File: templates/docs/map.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+          }).setView([20.5937, 78.9629], 5);
+
+          L.tileLayer('https://tile.openstreetmap.org/{z}/{x}/{y}.png', {
+            maxZoom: 19,
+            attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>'
+          }).addTo(map);
+
+          var rustaceanIcon = L.icon({
+              iconUrl: '/rustacean-flat-happy.png',
+              iconSize: [38, 38], // size of the icon
```
Comment (me-diru) [inline templates/docs/map.html:48]: please make it [45, 30] to make it look better on map

File: templates/docs/map.html [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+  </div>
+
+  <!-- <div class="content">
+    <div class="row flex-xl-nowrap">
+	  {{ macros_sidebar::docs_sidebar(current_section=current_section) }}
+	  {{ macros_toc::docs_toc(page=page) }}
+      <main class="docs-content col-lg-11 col-xl-9">
+      </main>
+    </div>
+  </div> -->
```
Comment (me-diru) [inline templates/docs/map.html:83]: please remove this if not used

**Review by me-diru — COMMENTED** [review]
this looks great, thanks!

please make the final changes requested and we can merge it :D

**Review by me-diru — APPROVED** [review]

File: templates/docs/map.html (hunk 4 above)
Comment (bharattech) [reply templates/docs/map.html]: Done

**Pull request merged**